biapol_utilities/label/_matching_algorithms.py

```diff
@@ -0,0 +1,259 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from biapol_utilities import utilities
+
+
+def max_similarity(label_image_x, label_image_y, similarity_matrix):
```

Comment:
If the function does some matching (that's what the documentation suggests), would it make sense to have "match" in the name?

Reply:
Changed in aca6738e336ac3fcb018f3188ebea025637c4ec8

Reply:
Functions are now called `match_something`, e.g., match_max_similarity